conanfile_stack.py

```diff
@@ -18,21 +18,23 @@ class matterRecipe(ConanFile):
     description = "Matter extension for Simplicity SDK Suite"
     license = "www.silabs.com/about-us/legal/master-software-license-agreement"
     author = "Silicon Laboratories Inc."
-    homepage = ""  # E.g.,: https://stash.silabs.com/projects/<space>/repos/<project>/browse/README.md
-    url = ""  # E.g.,: https://stash.silabs.com/projects/<space>/repos/<project>/browse
+    homepage = "https://github.com/SiliconLabsSoftware/matter_extension/blob/main/README.md"  # Ex: https://stash.silabs.com/projects/<space>/repos/<project>/browse/README.md
```

Comment:
all those settings are duplicated between both scripts.

A module could be made and used by both scripts to reduce duplication.